docs/t-sql/queries/from-using-pivot-and-unpivot.md

```diff
@@ -38,22 +38,43 @@ The syntax for `PIVOT` is simpler and more readable than the syntax that may oth
 The following syntax summarizes how to use the `PIVOT` operator.  
   
 ```syntaxsql  
-SELECT <non-pivoted column>,  
-    [first pivoted column] AS <column name>,  
-    [second pivoted column] AS <column name>,  
-    ...  
-    [last pivoted column] AS <column name>  
+SELECT [<non-pivoted column> AS <column name>,]
+    ...
+    <first pivoted column> AS <column name>,
+    <second pivoted column> AS <column name>,
+    ...
+    <last pivoted column> AS <column name>
```

Comment:
The original syntax here is correct. The first column is not optional and hence encased in `< >` to denote the non-pivoted column. The rest of the columns are optional, and should be encased by brackets `[ ]`

Reply:
> The original syntax here is correct. The first column is not optional and hence encased in < > to denote the non-pivoted column.

First, the first, non-pivoted column is optional. I can remove that from the first PIVOT example and when I run it, it works fine:

```-- Pivot table with one row and five columns  
SELECT [0], [1], [2], [3], [4]  
FROM  
(
  SELECT DaysToManufacture, StandardCost   
  FROM Production.Product
) AS SourceTable  
PIVOT  
(  
  AVG(StandardCost)  
  FOR DaysToManufacture IN ([0], [1], [2], [3], [4])  
) AS PivotTable; ```

So I believe using the "< >" to denote the column and "[ ]" to denote the whole thing, including "AS" and the alias, is correct. Technically, the AS clause is also optional, so it possibly should be:

> SELECT [<non-pivoted column> [AS <column name>], ]

> The rest of the columns are optional, and should be encased by brackets [ ]

If that's the case, then the whole line, including "AS" and the alias should be encased in the square brackets "[ ]". The characters around the column name, ex: currently "[first pivoted column]", denoting the column only, should be "<...>", as in "<first pivoted column>". Therefore, the syntax for that line would be:

> [<first pivoted column> [AS <column name>], ]

The remaining lines under the SELECT clause would be similar.

Reply:
Hi @Rob-S - Thanks! You're right, you can run the script without a non-pivoted column. But I don't see that as very useful if that's the case. I agree with your changes though.